src/site/artifacts/cargo_dist.rs

```diff
@@ -292,3 +291,16 @@ pub fn build_list(manifest: &DistManifest, config: &Config) -> Result<Box<div<St
     </div>
     ))
 }
+
+fn create_download_link(config: &Config, name: &String, version: Option<String>) -> String {
```

Comment:
version should just be a string IMO- unless there's a case where it is actually optional? (everything i've read in this diff so far manually adds `Some`)

Reply:
you are right, fixed!